docs/pages/api-docs/data-grid/grid-col-def.md

```diff
@@ -0,0 +1,42 @@
+| <span class="prop-name optional">filterable<sup><abbr title="optional">?</abbr></sup></span>                | <span class="prop-type">boolean</span>                                                                                                    | <span class="prop-default">true<br /></span>     | If `true`, the column is filterable.                                                                  |
+| <span class="prop-name optional">flex<sup><abbr title="optional">?</abbr></sup></span>                      | <span class="prop-type">number</span>                                                                                                     |                                                  | If set, it indicates that a column has fluid width. Range [0, ∞).                                     |
+| <span class="prop-name optional">headerAlign<sup><abbr title="optional">?</abbr></sup></span>               | <span class="prop-type">GridAlignment</span>                                                                                              |                                                  | Header cell element alignment.                                                                        |
+| <span class="prop-name optional">headerClassName<sup><abbr title="optional">?</abbr></sup></span>           | <span class="prop-type">string \| string[]</span>                                                                                         |                                                  | Class name that will be added in the column header cell.                                              |
+| <span class="prop-name optional">headerName<sup><abbr title="optional">?</abbr></sup></span>                | <span class="prop-type">string</span>                                                                                                     |                                                  | The title of the column rendered in the column header cell.                                           |
+| <span class="prop-name optional">hide<sup><abbr title="optional">?</abbr></sup></span>                      | <span class="prop-type">boolean</span>                                                                                                    | <span class="prop-default">false<br /></span>    | If `true`, hide the column.                                                                           |
+| <span class="prop-name optional">hideSortIcons<sup><abbr title="optional">?</abbr></sup></span>             | <span class="prop-type">boolean</span>                                                                                                    |                                                  | Toggle the visibility of the sort icons.                                                              |
```

Comment:
**Off topic**

Shouldn't all the boolean types have a default value?

```diff
diff --git a/packages/grid/_modules_/grid/models/colDef/gridColDef.ts b/packages/grid/_modules_/grid/models/colDef/gridColDef.ts
index ea855319..808aa13c 100644
--- a/packages/grid/_modules_/grid/models/colDef/gridColDef.ts
+++ b/packages/grid/_modules_/grid/models/colDef/gridColDef.ts
@@ -113,6 +113,7 @@ export interface GridColDef {
   headerAlign?: GridAlignment;
   /**
    * Toggle the visibility of the sort icons.
+   * @default false
    */
   hideSortIcons?: boolean;
   /**
```

Reply:
Fixed.